workflow/rules/functions.smk

```diff
@@ -65,7 +71,7 @@ def create_refcon_base_map():
 
 def _find_base_def_file(base_name):
 
-    repo_path = pl.Path(workflow.basedir).absolute()
+    repo_path = pl.Path(workflow.basedir).parent.resolve(strict=True)
```

Comment:
please add a `TODO` for the future here: this line needs to be changed when we adapt the refcon repo to the Snakemake template, and use a constant from the template

Reply:
TODO added